### PR TITLE
HelmRelease with spec change YAML

### PR DESCRIPTION
**Pull request by pezhang**
1. Add the YAML file with HelmRelease specifying spec field
2. This YAML file will be used for testing " A spec change update". It's related with https://github.com/open-cluster-management/multicloud-operators-subscription-release/pull/165. It changed the spec fields:   "replicaCount: 4" ->  "replicaCount: 3:

### CODE REVIEW

**Comment by pezhang** [comment]
@ianzhang366 Could you help to review this patch? Thanks a lot.

File: examples/test-guestbook-spec-change.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  name: guestbook
+  namespace: helmrelease-spec-test
+repo:
+  chartName: nginx-chart
+  source:
+    github:
+      branch: gh-pages
+      chartPath: nginx-chart
+      urls:
+        - https://github.com/pezhang/helm-release-API-test.git
```
Comment (mikeshng) [inline examples/test-guestbook-spec-change.yaml:19]: I don't think we should point at personal repo like this.

Could you not upload the charts you need to this repo first then reference this repo?

**Comment by pezhang** [comment]
@mikeshng Thanks a lot.

**Pull request merged**